test/test-checkout.sh

```diff
@@ -155,3 +155,94 @@ begin_test "checkout: outside git repository"
   grep "Not in a git repository" checkout.log
 )
 end_test
+
+begin_test "checkout: with include"
+(
+  set -e
+
+  reponame="checkout-with-include"
+  setup_remote_repo "$reponame"
+
+  clone_repo "$reponame" repo2
+
+  git lfs track "*.dat" 2>&1 | tee track.log
+  grep "Tracking \"\*.dat\"" track.log
```

Comment:
This pattern is also common in `test/test-checkout.sh`, but is not necessary and no longer conforms to our style. `git lfs track` alone (without `grep(1)` afterwords) is fine.